routes/comment.js

```diff
@@ -26,30 +26,18 @@ app.post('/', (request, response) => {
 
 // Deletes a comment from a post (USER)
 app.delete('/:id', async (request, response) => {
-  const commentId = request.params.id;
-  const comment = await Comment.fetchCommentId(commentId);
-  if (comment[0].user_id === request.user.id) {
-    Comment.removeComments(commentId)
```

Comment:
Do we need this user check? Will removing it allow anyone to delete any comment instead of just their own?

I think the rest is all set, looks very clean

Reply:
I think it's needed. Ignoring the FE-- without that code couldnt anyone delete anyone's comments with Postman?